src/components/widgets/LabelStatic.vue

```diff
@@ -13,16 +13,21 @@ defineProps<{
       <span class="top-label">
         {{ label }}
       </span>
-      <div :class="{ 'alert alert-warning': !modelValue }">
-        <span class="form-control">{{ modelValue }}</span>
+      <div class="top-left-corner">
+        <span 
+          class="form-control top-left-corner" 
+          :class="{ 'alert alert-warning': !modelValue }"
+        >
+          {{ modelValue }}
+        </span>
       </div>
     </div>
     <div v-else class="row">
       <span class="form-label col-3">
         {{ label }}
       </span>
-      <div class="col-9" :class="{ 'alert alert-warning': !modelValue }">
-        <span class="form-control">{{ modelValue }}</span>
+      <div class="col-9">
+        <span class="form-control" :class="{ 'alert alert-warning': !modelValue }">{{ modelValue }}</span>
```

Comment:
Good job on addressing this one! It was bugging my mind in my branch and was planning to address it afterwards :smile: